### DISCUSSION TITLE
added some more ratios

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2320] [Q&A] [answered]
i added some more ratios for like instagram etc and desktops

but i have one small error 1080x566 any ideas?

![screenshot-127 0 0 1_7865-2024 02 21-17_49_21](https://github.com/lllyasviel/Fooocus/assets/35565479/c03ec1f2-f52d-4efb-ab83-f4c6355d7b2d)

**Comment by ILikeToasters** [comment]
If you are talking about the 540:283 that is actually correct. It won't show fractional numbers so that is what it will show.

**Comment by mashb1t** [comment, marked as answer]
@lgwjames please find more information why these resolutions will not be added in https://github.com/lllyasviel/Fooocus/pull/1617
